docs/API.md

```diff
@@ -42,18 +42,15 @@ const app = express()
 
 app.use((req, res, next) => {
   const span = tracer.startSpan('web.request')
-  const scope = tracer.scopeManager().activate(span)
-
-  next()
+  
+  tracer.scope().activate(span, () => next())
 })
 
 app.get('/hello', (req, res, next) => {
   setTimeout(() => {
-    const scope = tracer.scopeManager().active() // the scope activated earlier
-    const span = scope.span() // the span wrapped by the scope
+    const span = tracer.scope().active() // the span activated earlier
```

Comment:
```suggestion
    const span = tracer.scope().active() // the `web.request` span activated earlier
```

Reply:
Just want to make sure this example is really easy for people to follow, no ambiguity

Reply:
Doc is being rewritten in another PR.